src/com/github/Kraken3/AFKPGC/BotDetector.java

```diff
@@ -391,5 +354,14 @@ public static void parseBanlist() {
 			AFKPGC.logger.warning("Error while trying to parse the banned players file");
 		}
 	}
+	
+	public static void warnPlayer(Player p) {
+		if (p != null) {
+			p.sendMessage("[WARNING] You were determined to be a cause of lag, if you dont stay out of the area "+
+					"you are in right now, you will be temporarily banned");
+			AFKPGC.debug(p.getUniqueId()," was informed that he is causing lag");

```

Comment:
Follow the AFKPGC standard; AFKPGC.debug(p.getUniqueId(), " (", p.getName(), ") was notified of presence in lag source.");